src/locales/de.yaml

```diff
@@ -0,0 +1,67 @@
+language: 🇩🇪 Deutsch
+
+service:
+  greeting: |
+    🖐 Hi, <b>${name}</b>! Ich kann den Hintergrund von jedem Foto entfernen, das du mir schickst. Probiere es aus und sende mir eine Datei oder ein Foto, um die Magie zu sehen.<a href="https://i.ibb.co/dKxZf5G/preview.png">ᅠ</a>
+
+    ⭐️ Ich kann dein Bild auch in einen Sticker umwandeln. Verwende den Befehl /settings, um diese Option zu aktivieren.
+  settings: ${beta_sign} In <b>${range}</b> hast du <b>${usage} image${plural} wovon <b>${converted_to_file} ${verb_file} in eine Datei</b> und <b>${converted_to_sticker} ${verb_sticker} in einen Sticker umgewandelt wurden</b>, erhalten.
```

Comment:
In <b>${range}</b> hast du <b>${usage} image${plural} wovon <b>${converted_to_file} in eine Datei</b> und <b>${converted_to_sticker} in einen Sticker umgewandelt wurden</b>, erhalten.